### ISSUE TITLE
Fix PHP Doc Blocks

### ISSUE THREAD

**Issue by DarkMukke** [usmanhalalit/pixie#150]
There seem to be many PHP Doc block incorrectly defined.

Eg : 
```
class QueryBuilderHandler
{
   ...
    /**
     * @var null|PDOStatement
     */
    protected $pdoStatement = null;

    ...
    /**
     * @param $tables Single table or multiple tables as an array or as
     *                multiple parameters
     *
     * @return static
     */
    public function table($tables)
    {
```

needs to be

```
class QueryBuilderHandler
{
   ...
    /**
     * @var null|\PDOStatement
     */
    protected $pdoStatement = null;

    ...
    /**
     * @param sting|array $tables Single table or multiple tables as an array or as
     *                multiple parameters
     *
     * @return static
     */
    public function table($tables)
    {
```

on a side note, in php 5.6 we should use splat vs func_get_args

**Comment by DarkMukke** [comment]
IF you want i can create a PR for that as I already fixed it locally

**Comment by TCB13** [comment]
@DarkMukke go ahead. Thank you.

**Comment by TCB13** [comment]
This make my #167 but since you've the code, just PR.

**Comment by DarkMukke** [comment]
The PR is not perfect though, it fails a test and I did this too long ago to remember why.
I had to re-fork and make some changes so it took me a bit to get around to do it.

**Comment by TCB13** [comment]
@DarkMukke I noticed there are other additional changes not related to the DocBlocks. What about just committing DocBlock changes right now and ignore everything else? 👍

**Comment by DarkMukke** [comment]
Right, that's how I had it in my local, Will make a small PR for just dock blocks

**Comment by TCB13** [comment]
@DarkMukke any news on the smaller just DocBlock PR?

**Comment by DarkMukke** [comment]
@TCB13 been on holiday (off-line), it's on my todo-list

**Comment by allejo** [comment]
Any updates on this?

**Comment by DarkMukke** [comment]
Oh woops, i have the commit, let make it a PR

**Comment by vikkio88** [comment]
this is fixed by #183

**Comment by vikkio88** [comment]
@DarkMukke you using phpstorm arent you? LoL I found this annoying to and fixed it while fixing another bug

**Comment by TCB13** [comment]
Yeah @vikkio88 it is true. @DarkMukke can you check if the changes by @vikkio88 at 220b920904b9f60a7d61eba9554d936f4fda7977 completely fix this? In my opinion they do.

**Comment by vikkio88** [comment]
I tested in my local env and they did. but is good to have another pair of eyes on it
